src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -28,6 +28,7 @@ import { UserProfileModal } from "components/organisms/UserProfileModal";
 import { WithNavigationBar } from "components/organisms/WithNavigationBar";
 
 import { AnnouncementMessage } from "components/molecules/AnnouncementMessage";
+import { PostersProvider } from "../../hooks/posters";
```

Comment:
This should be sorted with other `hooks/*` imports.

```suggestion
import { PostersProvider } from "hooks/posters";
```